.github/workflows/continuous-integration.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
+        # TODO switch to Golang 1.18 once https://github.com/golangci/golangci-lint/issues/2649 is fixed
         go-version: 1.17
```

Comment:
I was able to fix the issue by using temporarily go 1.17 for lint. 

once this PR is merged I will create an issue in this repo to track this.